internal/testutils/marshal_any.go

```diff
@@ -18,19 +18,20 @@
 package testutils
 
 import (
-	"fmt"
-
 	"github.com/golang/protobuf/proto"
-	"github.com/golang/protobuf/ptypes"
+	"google.golang.org/protobuf/protoadapt"
 	"google.golang.org/protobuf/types/known/anypb"
+	"testing"
 )
 
 // MarshalAny is a convenience function to marshal protobuf messages into any
 // protos. It will panic if the marshaling fails.
-func MarshalAny(m proto.Message) *anypb.Any {
-	a, err := ptypes.MarshalAny(m)
+func MarshalAny(t *testing.T, m proto.Message) *anypb.Any {
+	t.Helper() // Mark this function as a test helper.
+
+	a, err := anypb.New(protoadapt.MessageV2Of(m))
 	if err != nil {
-		panic(fmt.Sprintf("ptypes.MarshalAny(%+v) failed: %v", m, err))
+		t.Fatalf("ptypes.MarshalAny(%+v) failed: %v", m, err) // Use t.Fatalf instead of panic.
```

Comment:
Also, please change the error string.

`t.Fatalf("Failed to marshal proto %+v into an Any: %v", m, err)`